core/modules/contact/contact.admin.inc

```diff
@@ -5,6 +5,55 @@
  * Admin page callbacks for the Contact module.
  */
 
+/**
+ * Page callback: Contact settings.
```

Comment:
As a form, this shouldn't be described as a page callback. Simply "Contact settings form." is sufficient.